### PR TITLE
Text update for IETF 116

### LINKED ISSUE #39: Need to find TMF reference for the equipment document SD2-20_EquipmentModel.pdf
In our Inventory model we use as reference TMF document related to Equipment model but we have no official TMF reference to use in the draft in the References section.

### LINKED ISSUE #59: Number of authors (RFC7322)
According to RFC7322 the number of authors in the front page is limited to five people:

https://datatracker.ietf.org/doc/html/rfc7322#section-4.1.1

### PR DESCRIPTION
**Pull request by YuChaode**
Text update:
1. refresh the author list: fix #59
2. support navigation from topology to inventory: see #33
3. Updated TMF reference: fix #39
4. Updated references: see issue #58

Co-authored-by: italobusi [<italo.busi@huawei.com>](mailto:%3Citalo.busi@huawei.com%3E)

### CODE REVIEW

**Comment by italobusi** [comment]
[models-relationship-00.pptx](https://github.com/ietf-ccamp-wg/ietf-network-inventory/files/10922843/models-relationship-00.pptx)

File: draft-ietf-ccamp-network-inventory-yang.md [hunk 1]
```diff
@@ -562,9 +658,9 @@ sourcecode-markers="true" sourcecode-name="ietf-network-inventory@2022-07-11.yan
 
 ## Comparison With Openconfig-platform Data Model
 
-Since more and more devices can be managed by domain controller through OpenConfig, to avoid that our inventory data model cannot cover these devices' inventory data, we have compared our inventory data model with the openconfig-platform.yang which is the data model used to manage inventory information in OpenConfig.
+Since more and more devices can be managed by domain controller through OpenConfig, to avoid that our inventory data model cannot cover these devices' inventory data, we have compared our inventory data model with the "openconfig-platform" model which is the data model used to manage inventory information in OpenConfig.
```
Comment (jbouqui153) [inline draft-ietf-ccamp-network-inventory-yang.md:661]: better in positive way: "to **ensure** that our inventory data model **can** cover these devices' inventory data

File: draft-ietf-ccamp-network-inventory-yang.md [hunk 2]
```diff
@@ -609,9 +705,9 @@ Openconfig-platform data model is NE-level and uses a generic component concept
...
-{: #tab-oc title="Comparison between openconfig-platform and inventory data model"}
+{: #tab-oc title="Comparison between openconfig platform and inventory data models"}
 
-As it mentioned in {{reference-RFC8348}} that state data and performance data are out of scope of our data model, it is same for alarm data and it should be defined in some other alarm data models separately. And for some component specific structures in openconfig-platform, we consider some of them can be contained by our existing structure, such as fan, backplane, and controller-card. And for some of them, there is no need to manage for operators, such as storage and cpu.
+As it mentioned in {{reference-RFC8348}} that state data and performance data are out of scope of our data model, it is same for alarm data and it should be defined in some other alarm data models separately. And for some component specific structures in "openconfig-platform", we consider some of them can be contained by our existing structure, such as fan, backplane, and controller-card. And for some of them, there is no need to manage for operators, such as storage and cpu.
```
Comment (jbouqui153) [inline draft-ietf-ccamp-network-inventory-yang.md:710]: Remove "And for some of them, there is no need to manage for operators, such as storage and cpu" and replace by ", while some others do not need to be included in this network inventory model like storage and cpu"

File: ietf-hw-inventory-ref-topo.yang [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+      "Initial revision.";
+     reference
+       "RFC XXXX: A YANG Data Model for Network Hardware Inventory.";
+       //RFC Editor: replace XXXX with actual RFC number, update date
+       //information and remove this note
+  }
+
+  augment "/nw:networks/nw:network/nw:node" {
+    description 
+      "Information that allows the navigation between this node and
```
Comment (jbouqui153) [inline ietf-hw-inventory-ref-topo.yang:88]: that allows **a relationship** between

File: ietf-hw-inventory-ref-topo.yang [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+	  
+	  description
+      "The identifier of the Network Element (NE) from which this
+      node is abstracted";
+    }
+  }
+  
+  augment "/nw:networks/nw:network/nw:node/nt:termination-point" {
+    description 
+      "Information that allows the navigation between this Link
```
Comment (jbouqui153) [inline ietf-hw-inventory-ref-topo.yang:107]: that allows **a relationship** between this link

**Review by jbouqui153 — APPROVED** [review]

File: ietf-hw-inventory-ref-topo.yang [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+       //RFC Editor: replace XXXX with actual RFC number, update date
+       //information and remove this note
+  }
+
+  augment "/nw:networks/nw:network/nw:node" {
+    description 
+      "Information that allows the navigation between this node and
+      the Network Element (NE) from which this node is abstracted in
+      the network hardware inventory model.";
+
```
Comment (sergiobelotti) [inline ietf-hw-inventory-ref-topo.yang:91]: I would reshuffle a little the description:
" Information that allows the navigation between the node in the topology and
      the Network Element (NE)  in the network hardware inventory model from which the node is abstracted

File: ietf-hw-inventory-ref-topo.yang [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+      node is abstracted";
+    }
+  }
+  
+  augment "/nw:networks/nw:network/nw:node/nt:termination-point" {
+    description 
+      "Information that allows the navigation between this Link
+      Termination Point (LTP) and the port component from which this
+      LTP is abstracted in the network hardware inventory model.";
+
```
Comment (sergiobelotti) [inline ietf-hw-inventory-ref-topo.yang:110]: Also here I would say: 
"Information that allows the navigation between the Link Termination Point (LTP) and the port component in the network hardware inventory model from which this LTP is abstracted ."

**Review by sergiobelotti — APPROVED** [review]

**Pull request merged**